hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -1325,8 +1326,11 @@ public static boolean scmInit(OzoneConfiguration conf,
       // If SCM HA was not being used before pre-finalize, and is being used
       // when the cluster is pre-finalized for the SCM HA feature, init
       // should fail.
+      final LayoutVersionManager layoutVersionManager =
+          new HDDSLayoutVersionManager(scmStorageConfig.getLayoutVersion());
       ScmHAUnfinalizedStateValidationAction.checkScmHA(conf, scmStorageConfig,
-          new HDDSLayoutVersionManager(scmStorageConfig.getLayoutVersion()));
+          layoutVersionManager);
+      layoutVersionManager.close();
```

Comment:
nit: Let's wrap in `try-finally` to ensure it's closed even if exception is thrown (although the duplicate metrics won't be a problem in that case).

Reply:
Done!